app/services/task_status/validators/proceedings_types.rb

```diff
@@ -0,0 +1,34 @@
+module TaskStatus
+  module Validators
+    class ProceedingsTypes < Base
+      def valid?
+        return false if proceedings.empty?
+
+        super
+      end
+
+    private
+
+      delegate :proceedings, to: :application
+
+      def forms
+        [
+          client_involvement_type_form,
```

Comment:
be good to indicate it is plural/set/array here
e.g.
```suggestion
          client_involvement_type_forms,
          client_involvement_type_form_set,
          client_involvement_type_form_arr
```